project-boilerplates/reinforcement-learning/rl_boilerplate/main.py

```diff
@@ -1,14 +1,23 @@
 from rl_boilerplate import agent, environment
-
-from config import CFG
+from rl_boilerplate.config import CFG
```

Comment:
This was the issue. We need to import CFG the same way on each module to use python "cache import" mechanism and get the desired singleton behavior